### DISCUSSION TITLE
How can I increase the max file size when using react-quill

### DISCUSSION THREAD

**Discussion by pullmann4rent** [remix-run/remix#9606] [Q&A]
Hello,

I use https://github.com/zenoamaro/react-quill and I can there insert images on the editor.

But I got this error :

`MaxPartSizeExceededError: Field "description" exceeded upload size of 3000000 bytes.`

code:

```
  <TextEditor
    theme="snow"
    placeholder="Write description"
    onChange={setDesc}
    value={desc}
    className="editor-modal"
  />
```

How can I increase the max file size because I dont have control about the flow to upload the image when using react-quill.

Where can I change it now the max file size ?

**Comment by sergiodxa** [comment]
Is the size limit on React Quill? If it's on their side, then I recommend you to ask them, but I doubt the editor has a file size limit, instead it's probably a server-side limit on how you parse files.

If you're using Remix's [parseMultipartFormData](https://remix.run/docs/en/main/utils/parse-multipart-form-data) function then the upload handlers usually accept a `maxPartSize` option where you limit the size.
